consistent-sampling/src/main/java/io/opentelemetry/contrib/sampler/consistent/ConsistentReservoirSamplingSpanProcessor.java

```diff
@@ -55,7 +55,7 @@ private static final class ReadableSpanWithPriority {
     private final int rval;
     private final long priority;
 
-    public static ReadableSpanWithPriority create(
+    static ReadableSpanWithPriority create(
```

Comment:
cool, removing unnecessary public modifier and reducing since it is only used internally.